### DISCUSSION TITLE
Widget's Menu Assignment

### DISCUSSION THREAD

**Discussion by Banuk1616** [templaza/astroid-framework#729] [Q&A] [answered]
Could someone be so kind as to explain to me how this works? I don't understand how I can assign this to a page.

Quote beginning:
You can choose between All pages, No pages, Only on the pages selected from the list.

    On all pages: The widget is visible on all pages of your website

    No pages: The widget is invisible on all pages

    Only on the selected pages: “Menu selection” is displayed with all menu items. You can assign the widget by selecting the menu links with which you want the widget to appear on which pages.
End quote.

I only see these three options, but a small selection of the pages on which the widget should appear. I can't select a page from a list.
I'm feeling pretty stupid right now because I don't understand this.

Translated with DeepL.com (free version)

**Comment by sonvnn** [comment, marked as answer]
Hi @Banuk1616 

The mechanism works like Joomla module assignment. When you select "Only on the selected pages" it will display as image below.
![Screen Shot 2024-07-19 at 09 38 31](https://github.com/user-attachments/assets/97aa0ff4-4335-4a79-a7d0-e489ac0fa5f3)

If you can not see result like that. Please send me your admin account via sonlv@templaza.com
I will check it.

Thanks & Best Regards,
Sonny

**Reply by Banuk1616** [reply]
@sonvnn

I was able to solve the problem. After I reinstalled Joomla and Astroid on my local host I'm testing on, it works.
Many thanks and best regards Olli
